packages/fiori/src/UploadCollection.hbs

```diff
@@ -16,11 +16,10 @@
 		</ui5-list>
 		{{#if _showNoData}}
 			<div class="{{classes.noFiles}}">
-				<div class="icon-container">
-					<ui5-icon name="document"></ui5-icon>
-				</div>
-				<ui5-label class="title" wrapping-type="Normal">{{_noDataText}}</ui5-label>
-				<ui5-label class="subtitle" wrapping-type="Normal">{{_noDataDescription}}</ui5-label>
+				<ui5-illustrated-message name="Tent">
+					<ui5-title slot="title" class="title" level="H1">{{_noDataText}}</ui5-title>
```

Comment:
Change the title level to - level="H2", as a HTML page, only one "H1" is allowed.